### ISSUE TITLE
Updates to themeable properties with Firefox 89

### ISSUE THREAD

**Issue by caitmuenster** [mozilla/FirefoxColor#969]
We should update Firefox Color to reflect changes coming to themes for Firefox 89 (release date: June 1, 2021): 

* `tab_background_separator` will no longer be supported
* `toolbar_field_separator` will no longer be supported

We'll need to remove those properties from the Advanced Colors section of Firefox Color.

**Comment by Rob--W** [comment]
I'd be careful with removing them, as it could result in loss of data when e.g. building themes on Nightly for ESR78 (i.e. read theme > save theme > open theme = theme lost some colors).

We can consider marking them as obsolete/deprecated.

**Comment by caitmuenster** [comment]
Makes sense! Maybe that info could stay under the hood once 89 goes into release?

**Comment by Rob--W** [comment]
@caitmuenster Where is the bug/work that deprecates `tab_background_separator` ? The property is still supported in the current code (and not marked as deprecated, unlike `toolbar_field_separator`), but it is only effective when Proton is disabled.

- theme API: https://searchfox.org/mozilla-central/rev/4648b6ee31c2519b1753023e4f4853b14fdd16e5/toolkit/components/extensions/parent/ext-theme.js#204
- relation between API and CSS variable: https://searchfox.org/mozilla-central/rev/4648b6ee31c2519b1753023e4f4853b14fdd16e5/browser/modules/ThemeVariableMap.jsm#51-53
- CSS usage with proton condition: https://searchfox.org/mozilla-central/rev/4648b6ee31c2519b1753023e4f4853b14fdd16e5/browser/themes/shared/tabs.inc.css#959-961
- Bug that introduced the change: https://bugzilla.mozilla.org/show_bug.cgi?id=1681421

**Comment by Rob--W** [comment]
About the above question - that doesn't block this work in the Color repo, but if there is no existing effort a new bug should be filed to add the `"deprecated"` key to https://searchfox.org/mozilla-central/rev/4648b6ee31c2519b1753023e4f4853b14fdd16e5/toolkit/components/extensions/schemas/theme.json#131

It's just a oneline change, if you want to make the commit I will review it ;)

**Comment by caitmuenster** [comment]
Hm, I'm not seeing a bug for `tab_background_separator` -- I got that info from @msujaws. Jared, do you know where this is filed? 

> It's just a oneline change, if you want to make the commit I will review it ;)

Haha I have never committed code to Firefox before but I'm open to giving it a try! 😅 (/me runs for https://wiki.mozilla.org/WebExtensions/Contribution_Onramp)

**Comment by msujaws** [comment]
> Hm, I'm not seeing a bug for `tab_background_separator` -- I got that info from @msujaws. Jared, do you know where this is filed?

I'm not aware of a bug on file to mark this as "deprecated".

**Comment by caitmuenster** [comment]
Thanks! I can file one.

**Comment by caitmuenster** [comment]
Just filed https://bugzilla.mozilla.org/show_bug.cgi?id=1708213.
